ubuntudesign/www.ubuntu.com
# Cloud overview

## Done

Updated content and structure of /cloud page. Updated the pie chart generation and vimeo styling.

## QA

Check `/cloud` in all viewports and check the [copy](https://docs.google.com/document/d/1mQjVXDcoP8ChOzRdmuI55BdXhbURD53qBNc2bgJhfhg/edit#heading=h.561g5td2vnk3)
